app/v1/connectors/ListPeriodSummariesConnector.scala

```diff
@@ -39,7 +39,7 @@ class ListPeriodSummariesConnector @Inject() (val http: HttpClient, val appConfi
     val taxYear    = request.taxYear
     val downstreamUri =
       taxYear match {
-        case Some(taxYear) if taxYear.useTaxYearSpecificApi =>
+        case Some(taxYear) if taxYear.isTys =>
```

Comment:
np with the name change (less typing!), but we probably want to be wary of making common code too different across all the APIs

Reply:
I went for the rename as the original wording didn't make much sense outside of the context of the connector. The shortness is an added bonus! If you think it's an improvement, we can communicate shifting to this name for future 'Prepare API tickets', but otherwise happy to revert to keep it consistent.

Reply:
In cis-deductions I added this:
```
def isTys(implicit featureSwitches: FeatureSwitches): Boolean = useTaxYearSpecificApi
```
as 'useTaxYearSpecificApi' is more in the context of "what downstream route should I follow here" (not just "is this a TYS tax year"). I'm also happy to just rename it to isTys, but it ought to be renamed everywhere if so.  (Not straightaway ofc, but just to keep in mind!)

(It's also to make the transition easier later when we get the go-ahead to use a common MTD library)

Reply:
If you're happy with it, I'd prefer renaming over adding a second method. It doesn't seem right that the `TaxYear` class should have such a connector specific method name.